src/DynamoCoreWpf/Views/HomePage/HomePage.xaml.cs

```diff
@@ -0,0 +1,572 @@
+        {
+            GuidedTourItems = new List<GuidedTourItem>
+            {
+                new GuidedTourItem(Wpf.Properties.Resources.GetStartedGuide.TrimStart('_'),
+                    Wpf.Properties.Resources.GetStartedGuideDescription, GuidedTourType.UserInterface.ToString()),
+                new GuidedTourItem(Wpf.Properties.Resources.OnboardingGuide.TrimStart('_'),
+                    Wpf.Properties.Resources.OnboardingGuideDescription, GuidedTourType.GetStarted.ToString()),
+                new GuidedTourItem(Wpf.Properties.Resources.PackagesGuide.TrimStart('_'),
+                    Wpf.Properties.Resources.PackagesGuideDescription, GuidedTourType.Packages.ToString())
+            };
+        }
+
+        private void OnDataContextChanged(object sender, DependencyPropertyChangedEventArgs e)
+        {
+            startPage = this.DataContext as StartPageViewModel;
+                
+            if (startPage != null)
+            {
+                startPage.DynamoViewModel.PropertyChanged += DynamoViewModel_PropertyChanged;
+            }
+        }
+
+        private void DynamoViewModel_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if(webView?.CoreWebView2 != null && e.PropertyName.Equals(nameof(startPage.DynamoViewModel.ShowStartPage)))
+            {
+                webView.CoreWebView2.ExecuteScriptAsync(@$"window.setShowStartPageChanged('{startPage.DynamoViewModel.ShowStartPage}')");
+            }
+        }
+
+        /// <summary>
+        /// This is used before DynamoModel initialization specifically to get user data dir
+        /// </summary>
+        /// <returns></returns>
+        private string GetUserDirectory()
+        {
+            var version = AssemblyHelper.GetDynamoVersion();
+
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+            return Path.Combine(Path.Combine(folder, "Dynamo", "Dynamo Core"),
+                            String.Format("{0}.{1}", version.Major, version.Minor));
+        }
+
+        private async void UserControl_Loaded(object sender, System.Windows.RoutedEventArgs e)
+        {
+            string htmlString = string.Empty;   
+            string jsonString = string.Empty;
+
+            // When executing Dynamo as Sandbox or inside any host like Revit, FormIt, Civil3D the WebView2 cache folder will be located in the AppData folder
+            var userDataDir = new DirectoryInfo(GetUserDirectory());
+            PathHelper.CreateFolderIfNotExist(userDataDir.ToString());
+            var webBrowserUserDataFolder = userDataDir.Exists ? userDataDir : null;
+
+            webView.CreationProperties = new CoreWebView2CreationProperties
+            {
+                UserDataFolder = webBrowserUserDataFolder.FullName
+            };
+
+            //ContentRendered ensures that the webview2 component is visible.
+            await webView.EnsureCoreWebView2Async();
```

Comment:
use DynamoWebView2.Initialize() instead of EnsureCoreWebView2Async

Reply:
Same here, all working great.